### ISSUE TITLE
Does the architecture need to use HTTP?

### ISSUE THREAD

**Issue by martinthomson** [ietf-wg-privacypass/base-drafts#323]
The interaction model here doesn't really need to use HTTP as far as I can tell.  Obviously, the specific protocols need to be more concrete, but the architecture works without specifically saying that the interactions use HTTP.

The overview specifically mentions HTTP only with respect to the Client-Origin interactions, not other interactions, but I can't see how this is necessary.

**Comment by tfpauly** [comment]
No, it doesn't need to use HTTP. The auth scheme document that instantiates the challenges and redemption does use HTTP, but other mechanisms can be used.

**Comment by chris-wood** [comment]
I'm OK de-emphasizing HTTP in the architecture if it helps simplify the presentation. Beyond it overly restrictive, was there another reason that led to this issue, @martinthomson?

**Comment by martinthomson** [comment]
Only that I know that this has obvious applications beyond HTTP and the architecture has no direct need to use it.
